examples/solvers/pdhg_denoising_ROF_algorithm_comparison.py

```diff
@@ -75,7 +75,7 @@ def __call__(self, x):
         k = self.iteration_count
 
         self.iteration_counts.append(self.iteration_count)
-        self.ergodic_iterate = (k-1)/k * self.ergodic_iterate + 1/k * x
+        self.ergodic_iterate = (k - 1) / k * self.ergodic_iterate + 1 / k * x
```

Comment:
Didn't see this at first glance, but I think for this to work with Python 2 we need `from __future__ import division`.